data/formats-data.js

```diff
@@ -21,9 +13,6 @@ let BattleFormatsData = {
 	},
 	venusaur: {
 		randomBattleMoves: ["gigadrain", "leechseed", "sleeppowder", "sludgebomb", "substitute"],
-		eventPokemon: [
-			{"generation": 6, "level": 100, "isHidden": true, "moves": ["solarbeam", "frenzyplant", "synthesis", "grasspledge"], "pokeball": "cherishball"},
-		],
 		tier: "UU",
```

Comment:
In the future, I want `tier` to be an array of tiers, instead of a single tier, so we don't have the LC Uber issue.

Reply:
(But that's out of scope for this PR.)